src/huntsman/pocs/tests/test_states.py

```diff
@@ -35,8 +37,19 @@ def observatory(mount, cameras, images_dir):
 
 
 @pytest.fixture(scope='function')
-def pocs(observatory):
+def dome():
+    set_config('dome', {
+        'brand': 'Simulacrum',
+        'driver': 'simulator',
+    })
+    return create_dome_simulator()
+
+
+@pytest.fixture(scope='function')
+def pocs(observatory, dome):
+    os.environ['POCSTIME'] = '2020-01-01 08:00:00'
     pocs = POCS(observatory, run_once=True)
+    pocs.observatory.set_dome(dome)
```

Comment:
I'm not sure why we are using `set_dome` here. Surely we want to pass it to the observatory instance on creation like `HuntsmanObservatory(dome=dome, ...)` ?

Reply:
Well I checked it but it does not work. It will not create the dome atribute by that command

Reply:
Why didn't it work? What is the error? @bazkiaei

Reply:
It looks like it does not create the dome attribute that way. When I run the test and try to check the dome status, it gives the following error:

```
   assert not pocs.observatory.dome.is_open
E AttributeError: 'NoneType' object has no attribute 'is_open'
```

Reply:
@bazkiaei I think that implies something else is going wrong with the dome simulator creation. Can you verify `create_dome_simulator` is working here?

Reply:
@lspitler I checked `create_dome_simulator` and it is working here. The output of that is `class 'panoptes.pocs.dome.simulator.Dome'`, which is what we want. Following @danjampro advice, I keep `pocs.observatory.set_dome(dome)` and open an issue in POCS.

Reply:
sounds good @bazkiaei

Reply:
Can you make a new in the POCS issue linking to this PR? Want to probably change the call here once POCS is fixed.